.gitignore

```diff
@@ -1,7 +1,10 @@
 node_modules
+build
+dist
```

Comment:
I'd prefer to keep `dist` for build output: it's the same convention most packages follow, and all the other LUME packages follow.

Building locally can go into `dist/`, and on CI the same exact steps put the same stuff into `dist/`.